nvflare/private/fed/utils/fed_utils.py

```diff
@@ -391,3 +391,11 @@ def get_return_code(process, job_id, workspace, logger):
 
 def get_simulator_app_root(simulator_root, site_name):
     return os.path.join(simulator_root, site_name, SimulatorConstants.JOB_NAME, "app_" + site_name)
+
+
+def add_custom_dir_to_path(app_custom_folder, new_env):
```

Comment:
I prefer `add_custom_dir_to_python_path` for more clarity, might be good to add docstring and typehint